packages/router-component-store/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "An Angular Router-connecting NgRx component store.",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "^15.0.0",
-    "@angular/router": "^15.0.0",
-    "@ngrx/component-store": "^15.0.0",
+    "@angular/core": "^15.0.0 || ^16.0.0",
```

Comment:
Can confirm it DOES work in 16, we have been using it for a while.

Reply:
I have recently put it into a big Angular 16 app with complex routing configurations and route state usage and I can also confirm this. However, Router Component Store will follow the versioning scheme recommended by Angular: One major version of Router Component Store for every major version of Angular and NgRx Component Store.